misc/validate_accounting.sh

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env bash
```

Comment:
For reference only. This script works fine but it's too slow to run against all mainnet accounts.